src/FSharp.Data.GraphQL.Shared/Serialization/JsonConverters.fs

```diff
@@ -18,7 +19,7 @@ type ClientMessageConverter () =
     let invalidMsg (explanation : string) = InvalidMessage (4400, explanation) |> Result.Error
 
     let errMsgToStr (struct (docId : int, graphQLErrorMsgs : GQLProblemDetails list)) =
-        String.Join ('\n', graphQLErrorMsgs |> Seq.map (fun err -> err.Message))
+        String.Join ("n", graphQLErrorMsgs |> Seq.map (fun err -> err.Message))
```

Comment:
Apparently there's no `Join` overload for using a character as separator when targeting .netstandard2.0.

Reply:
Ah, yes. However the slash disappeared
```suggestion
        String.Join ("\n", graphQLErrorMsgs |> Seq.map (fun err -> err.Message))
```